File: UVa-Online-Judge/11764 - Jumping Mario.py
def main():
	from sys import stdin, stdout

	t = int(stdin.readline())
	tc = 1

	while tc <= t:
		stdin.readline()		# Just to skip, n is not necessary in this solution.
		walls = stdin.readline().split()
		
		prev = int(walls.pop(0))

		hiCnt, loCnt = 0, 0

		for h in walls:
			curr = int(h)

			if prev < curr: hiCnt += 1
			elif prev > curr: loCnt +=1

			prev = curr

		stdout.write("Case %d: %d %d\n" % (tc, hiCnt, loCnt))
		tc += 1

if __name__ == '__main__':
	main()

File: UVa-Online-Judge/11764 - Jumping Mario.c
#include <stdio.h>

int main()
{
    int t, tc, n, prev, curr, hiCnt, loCnt;

    scanf("%d", &t);

    for(tc = 1; tc <= t; ++tc) {
        scanf("%d %d", &n, &prev);

        hiCnt = loCnt = 0;

        while(--n) {
            scanf("%d", &curr);

            if(prev < curr) ++hiCnt;
            else if(prev > curr) ++loCnt;

            prev = curr;
        }

        printf("Case %d: %d %d\n", tc, hiCnt, loCnt);
    }

    return 0;
}
